### ISSUE TITLE
[Remoto] Computer Vision Engineer at AdMobilize

### ISSUE THREAD

**Issue by di3goleite** [devfsa/vagas#85]
## Job Description

We're looking for a very passionate developer, with senior experience in Computer Vision. If you like challenges and have a willingness to learn and share knowledge this job is for you. Those are the main responsibilities:

- Collaborate with the Computer Vision team to research, develop and evaluate innovative CV solutions.
- Turn research papers into implementations.
- Apply state of the art computer vision techniques (deep learning, and other traditional techniques) to solve challenging problems in object detection, recognition and pose estimation, and improve AdMobilize’s computer vision models and algorithms.
- Define trade offs between speed and accuracy, considering latency for the system as a whole.
- Build and sample datasets for training/evaluating your models.

## Location

Feira de Santana - Bahia or Remote

## Benefits

- Flexible Schedule
- Payment set in dollars
- Growing company, you'll grow with us.

## Requirements

- **3-5 years minimum** experience working on Computer Vision or related fields.
- **3+ years** of production-level development experience with C++ and Python.
- **2+ years** of experience with deep learning frameworks like Tensorflow, Keras, etc.
- Experience in Neural network compression/ Neural network optimization.
- Experience applying Machine Learning in imagining problems, such as object detection/segmentation and recognition
- Experience with OpenCV.
- Analytic mindset.
- Background in math and algorithm optimization is a plus.
- Strong leadership, communication and collaboration skills.
- Professional level English knowledge

## Our company

We're global team of 40 determined to give everyone in the world access to building technology-driven solutions for challenges they and others face. We speak over 10 languages and we're revolutionizing the way Out-Of-Home Advertising works through of digitalizing the physical world and bringing to advertisers analytical data so they can take better decision and have the best return of investment.

## How to apply

Please send a email with you resume to careers@admobilize.com with the subject: Computer Vision Engineer Job
